dataland-frontend/src/components/resources/dataTable/conversion/DataPoints.ts

```diff
@@ -139,3 +133,37 @@ function doesAnyDataPointPropertyExist(dataPointProperties: DatapointProperties
       (dataPointProperties?.dataSource && dataPointProperties?.dataSource.fileReference.trim().length > 0))
   );
 }
+
+/**
+ * Builds the displayValue object for the DataPointWrapperDisplayComponent when quality, comment and data source are NOT
+ * all empty.
+ * @param inputValue the original value to wrap
+ * @param fieldLabel the label of the field to wrap
+ * @param datapointProperties the properties of the datapoint to wrap
+ * @returns the built displayValue object
+ */
+function buildDisplayValueWhenDataPointMetaInfoIsAvailable(
+  inputValue: AvailableMLDTDisplayObjectTypes,
+  fieldLabel: string,
+  datapointProperties: DatapointProperties | undefined | null
+): MLDTDisplayComponentTypes[MLDTDisplayComponentName.DataPointWrapperDisplayComponent] {
+  let innerContent: AvailableMLDTDisplayObjectTypes;
+  const isOnlyQualityProvided =
+    datapointProperties?.dataSource == undefined &&
+    datapointProperties?.comment == undefined &&
+    inputValue.displayValue == '';
+
+  if (isOnlyQualityProvided) {
```

Comment:
If I understand the code correctly, neither quality has to be provided. Consider
```suggestion
  if (isAtMostQualityProvided) {
```